src/participant/Main.java

```diff
@@ -16,14 +16,14 @@ public static void main ( String[] args ) {
         for (Participant participant : participants) {
```

Comment:
This reminds a competition, isn't it? Please create a new class `Competition` and move this logic there under the method `start`.